examples/pipelines/hf_dataset_pipeline/components/load_from_hub/src/main.py

```diff
@@ -14,6 +13,17 @@
 logger = logging.getLogger(__name__)
 
 
+def create_metadata(batch):
```

Comment:
This would be functionality to add to Fondant as well once we have a fixed taxonomy.